packages/web-components/src/mixins/form-field-error.css

```diff
@@ -1,10 +1,16 @@
-#error-message, #input-error-message {
+#error-message {
     color: var(--color-secondary-dark);
     display: block;
     font-weight: 700;
     margin-bottom: 1.2rem;
 }
 
+#input-error-message {
+  color: var(--color-secondary-dark);
+  display: block;
+  font-weight: 700;
+}
+
```

Comment:
There is shared CSS here for the error message id's. It can be simplified to:

```css
#error-message {
    margin-bottom: 1.2rem;
}

#error-message, #input-error-message {
  color: var(--color-secondary-dark);
  display: block;
  font-weight: 700;
}
```